### PR TITLE
migration to jupyter book 0.14

### PR DESCRIPTION
**Pull request by gtduncan**
## Describe your changes
Migrates sklearn-evaluation to jupyter-book 0.14 (https://github.com/ploomber/contributing/issues/35)

## Issue ticket number and link
Closes x

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://sklearn-evaluation--318.org.readthedocs.build/en/318/

### CODE REVIEW

**Comment by gtduncan** [comment]
Docs seem to be failing on cell `NotebookCollection.md`. An ImportError is being raised for `lxml`— odd because it looks like it's a dependency in `environment.yml`. Didn't have this issue locally

**Review by edublancas — CHANGES_REQUESTED** [review]
we have some environment mismatch here

readthedocs is using doc/requirements.txt (which is missing lxml). try adding lxml to doc/requirements.txt, if it doesn't work we can add a doc/environment.yml but we'll also have to configure readthedocs to use it instead of the requirements, check out jupysql's config for an [example](https://github.com/ploomber/jupysql/blob/master/.readthedocs.yml)

(environment.yml is used by binder)

**Comment by gtduncan** [comment]
All good now!

File: environment.yml [hunk 1]
```diff
@@ -17,4 +17,5 @@ dependencies:
     - papermill
     - seaborn
     - ipywidgets
-    - jupysql
\ No newline at end of file
+    - jupysql
+    - jupyter-book
```
Comment (edublancas) [inline environment.yml:21]: do we need jupyter-book here? this is the environment for Binder

**Review by edublancas — CHANGES_REQUESTED** [review]

File: environment.yml (hunk 1 above)
Comment (gtduncan) [reply environment.yml]: Oops, my mistake

**Review by edublancas — CHANGES_REQUESTED** [review]
I noticed that Jupyter-book introduced dark mode, and the label isn't visible:

image

let's add some white border, so it's also visible in such case.

**Review by edublancas — APPROVED** [review]

**Comment by edublancas** [comment]
merged this now but opened another issue to tackle the image problem:  #323

**Pull request merged**